raddb/certs/server.cnf

```diff
@@ -7,10 +7,10 @@ certs			= $dir
 crl_dir			= $dir/crl
 database		= $dir/index.txt
 new_certs_dir		= $dir
-certificate		= $dir/server.pem
+certificate		= $dir/eapsvr.pem
 serial			= $dir/serial
 crl			= $dir/crl.pem
-private_key		= $dir/server.key
+private_key		= $dir/eapsvrkey.key
```

Comment:
And this ?

Keep in mind that you're adding functionnalities and that the rest of the system needs to work without your feature